tests/system/providers/google/cloud/gcs/example_gcs_sensor.py

```diff
@@ -107,72 +108,77 @@ def mode_setter(self, value):
 
     # [START howto_sensor_object_update_exists_task]
     gcs_update_object_exists = GCSObjectUpdateSensor(
-        bucket=BUCKET_NAME,
+        bucket=DESTINATION_BUCKET_NAME,
         object=FILE_NAME,
         task_id="gcs_object_update_sensor_task",
     )
     # [END howto_sensor_object_update_exists_task]
 
     # [START howto_sensor_object_update_exists_task_async]
     gcs_update_object_exists_async = GCSObjectUpdateSensor(
-        bucket=BUCKET_NAME, object=FILE_NAME, task_id="gcs_object_update_sensor_task_async", deferrable=True
+        bucket=DESTINATION_BUCKET_NAME,
+        object=FILE_NAME,
+        task_id="gcs_object_update_sensor_task_async",
+        deferrable=True,
     )
     # [END howto_sensor_object_update_exists_task_async]
 
-    upload_file = LocalFilesystemToGCSOperator(
-        task_id="upload_file",
-        src=UPLOAD_FILE_PATH,
-        dst=FILE_NAME,
-        bucket=BUCKET_NAME,
+    copy_file = GCSToGCSOperator(
```

Comment:
is this file always available in src gas bucket?

Reply:
> is this file always available in src GCS bucket?

Yes it is.
Actually you can check it in GCS bucket `airflow-system-tests-resources` the folder `gcs`